LocUpdFgService Updated

fullmetaltac opened the pull request:
LocUpdFgService Updated

pjcollins commented on android-o/AndroidPlayLocation/LocUpdFgService/LocUpdFgService/LocationUpdatesService.cs:
```diff
@@ -109,12 +114,20 @@ public override void OnCreate()
+
+		    if (Build.VERSION.SdkInt >= Build.VERSION_CODES.O)
+		    {
+		        var name = (String)GetString(Resource.String.app_name);
```
This cast is throwing on startup, we'll need to update it.

fullmetaltac replied:
Done.

fullmetaltac commented:
I fixed problem with runtime error on my device.
![image](https://user-images.githubusercontent.com/5435980/34152973-0cfeed0c-e4b9-11e7-8a51-e734840fa3c6.png)

fullmetaltac commented:
I was receiving error that described on stackoverflow:
https://stackoverflow.com/questions/41163887/could-not-resolve-type-with-token-0100000f
So I cleaned solution and restored nuget packages.
Changed target and minimum version for my phone.
After that service was working correctly.

pjcollins commented:
Replaced by #252
